ui/src/shared/apis/metaQuery.js

```diff
@@ -36,14 +36,14 @@ export function showMeasurements(source, db) {
 }
 
 export function showTagKeys({source, database, retentionPolicy, measurement}) {
-  const query = `SHOW TAG KEYS FROM "${measurement}"`
+  const query = `SHOW TAG KEYS FROM "${retentionPolicy}"."${measurement}"`
 
   return proxy({source, db: database, rp: retentionPolicy, query})
 }
 
 export function showTagValues({source, database, retentionPolicy, measurement, tagKeys}) {
   const keys = tagKeys.sort().map((k) => `"${k}"`).join(', ')
-  const query = `SHOW TAG VALUES FROM "${measurement}" WITH KEY IN (${keys})`
+  const query = `SHOW TAG VALUES FROM "${retentionPolicy}"."${measurement}" WITH KEY IN (${keys})`
```

Comment:
Ok! @lukevmorris @jademcgough @timraymond I've added checks for null RP here:

https://github.com/influxdata/chronograf/pull/1182/commits/e3c07e1797872fcdb6233e2676d8a4caa3dbde80